src/servlets/metaTags/index.ts

```diff
@@ -26,7 +25,7 @@ import {
 } from '../utils/helpers'
 import { Context, MetaTagFormat, Playable } from './types'
 
-const CAN_EMBED_USER_AGENT_REGEX = /(twitter|discord)/
+const CAN_EMBED_USER_AGENT_REGEX = /(twitter|discord|reddit|iframely)/
```

Comment:
This is a shot in the dark at what reddit unfurls will look like - I think they're using the provider.json from the OEmbed website so we'll have to add a pr there before they even attempt to use our oembeds